### DISCUSSION TITLE
Screw size under the head cover of HSR-C

### DISCUSSION THREAD

**Discussion by jjnak** [HumanSupportRobot/QandA#85] [Q&A] [answered]
I would like to know the size (length and diameter) of the screw that can fit the holes on the metal plate (there are 6 holes) of the HSR head under the cover (that is attached with magnets). 

Your help would be greatly appreciated.

-Jun

**Comment by yutaro-ishida** [comment, marked as answer]
Hi Jun,

The holes are
- M3
- pitch: 0.5
- length: 2 mm

Yuta

**Reply by jjnak** [reply]
Dear Yuta,

Thank you for providing the information.

-Jun
